app/ui/src/app/integration/edit-page/action-configure/action-configure.component.html

```diff
@@ -64,12 +64,12 @@ <h3>{{ action?.name }}</h3>
       </div>
       <syndesis-loading [loading]="loading">
         <ng-container *ngIf="error">
-          <div [class]="error.class"
```

Comment:
We need to keep this CSS class setter, at least somewhere, as this is set dynamically by the component both for this case and if there's an error while fetching the descriptor.  In the case where there's no configuration it's set to `alert alert-info`.

Reply:
@gashcrumb My bad, seemed like an easy enough change but I should have touched base first. Added it back to the element that also contains the output (`error.message`).

Reply:
I’m surprised this actually worked. The correct attribute should be [ngClass] instead.

Reply:
`[class]` works too, it's less powerful than `ngClass` but suits it's purpose here...

Reply:
The reason why it works is because it's part of the W3C standard and whatever binding we inject in a native attribute wrapped by brackets will get properly digested and interpolated. In any event, sticking to the Angular spec is the safest way to avoid weird bugs in the future: `[ngClass]` behavior is already tested by angular, while `[class]` don't.